src/repositoryModals.jsx

```diff
@@ -42,6 +43,34 @@ import * as remotes from './remotes';
 
 const _ = cockpit.gettext;
 
+const RemoteSelect = ({ remotes, selectedRemote, setSelectedRemote }) => {
+    const [isSelectOpen, setSelectOpen] = useState(false);
+
+    const toggle = toggleRef => (
+        <MenuToggle
+           ref={toggleRef}
+            onClick={() => setSelectOpen(!isSelectOpen)}
```

Comment:
Indentation  -- why do we have all these fancy linters if they don't spot this?

Reply:
Yeah good question, I can look into this but that might change it everywhere I suppose.